roles/dnsmasq/templates/network.conf.j2

```diff
@@ -0,0 +1,14 @@
+# Managed by ci-framework/dnsmasq
+{% for range in cifmw_dnsmasq_network_definition['ranges']   %}
+{%   if range.start_v6 is defined                            %}
+dhcp-range={{ range.label }},{{ range.start_v6 }},{{ range.end_v6 }},{{ range.netmask | default('64') }},{{ range.ttl | default('1h') }},static
```

Comment:
hmm, `range.netmask` - we cannot have a single var for both v4 and v6.

Should we name it `range.prefix_length_v4` and `range.prefix_length_v6` instead and do the calculus for IPv4 netmask? (I know the names are very long - but that would be the same used in network-mapper [output](https://github.com/openstack-k8s-operators/ci-framework/blob/main/tests/unit/module_utils/test_files/networking-definition-valid-all-tools-dual-stack-full-map-out.json#L276C12-L276C28).

`{{ (range.start_v4 + "/" + range.prefix_length_v4 | default(24) | string) |  ansible.utils.ipaddr('netmask') }}` 

`{{ range.prefix_length_v4 | default(64)  }}`

Reply:
good idea